lab5/Pasichnichenko_Task5_1.py

```diff
@@ -0,0 +1,17 @@
+from math import factorial
```

Comment:
Please consider some pieces of code:

from itertools import takewhile, count, repeat

row = (1 / pow(2, n) + 1 / pow(3, n) for n in count())
print(sum(list(takewhile(lambda x: x >= 1e-4, row))))

Reply:
from itertools import takewhile, count, repeat

row = ((i, n) for i, n in enumerate(repeat(n)))
print(len(list(takewhile(lambda x: x[1] // (10 ** x[0]) > 0, row))))